tasks/__init__.py

```diff
@@ -26,6 +26,12 @@
 NO_TESTS_COLLECTED = 5
 ns = Collection()
 
+try:
+    from tasks import local as local_tasks
```

Comment:
Are you sure this import works? Maybe you can help me with what I'm doing wrong - 

calling something like `docker-compose run --rm web invoke example_task HELLO`... and getting `No idea what 'example_task' is!` If I move `example_task` into this file, it works fine.

Reply:
there's a suffix so for example task it's `local.example_task' just enter `invoke` for a full list of tasks.

Reply:
John reminded me that I need to copy the local-dist.py to local -